src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -1,1304 +0,0 @@
-    ObaRoute obaRoute;
-
-    @Mocked
-    ObaRoute obaRoute2;
-
-    @Mocked
-    ObaStopResponse obaStopResponse;
-
-    @Mocked
-    ObaArrivalInfoResponse obaArrivalInfoResponse;
-
-    @Mocked
-    GoogleMaps googleMaps;
-
-    @Mocked
-    ObaClient obaClient;
-
-    @Mocked
-    ObaUserClient obaUserClient;
-
-    @Mocked
-    ObaDao obaDao;
-
-    @Mocked
-    ArrayList<ObaRegion> obaRegions;
-
-    @Resource
-    AuthedSpeechlet authedSpeechlet;
-
-    @Resource
-    ObaUserDataItem testUserData;
-
-    @Before
-    public void initializeAuthedSpeechlet() throws URISyntaxException {
-        authedSpeechlet.setUserData(testUserData);
-    }
-
-    @Before
-    public void resetFactoryObjects() {
-        session = Session.builder().withUser(testUser).withSessionId("test-session-id").build();
-    }
-
-    @Test
-    public void getStopDetails(@Mocked ObaStopResponse mockResponse) throws SpeechletException, IOException, URISyntaxException {
-        new Expectations() {{
-            mockResponse.getStopCode(); result = "6497";
-            mockResponse.getName(); result = "University Area Transit Center";
-            obaUserClient.getStopDetails(anyString); result = mockResponse;
-        }};
-
-        SpeechletResponse sr = authedSpeechlet.onIntent(
-                IntentRequest.builder()
-                        .withRequestId("test-request-id")
-                        .withIntent(
-                                Intent.builder()
-                                        .withName(GET_STOP_NUMBER)
-                                        .withSlots(new HashMap<String, Slot>())
-                                        .build()
-                        )
-                        .build(),
-                session
-        );
-
-        String spoken = ((PlainTextOutputSpeech)sr.getOutputSpeech()).getText();
-        assertThat(spoken, equalTo("Your stop is 6497, University Area Transit Center."));
-    }
-
-    @Test
-    public void launchTellsArrivals() throws SpeechletException, IOException {
-        // Turn off tutorials
-        session.setAttribute(ANNOUNCED_INTRODUCTION, 1L);
-        session.setAttribute(ANNOUNCED_FEATURES_V1_1_0, 1L);
-
-        ObaArrivalInfo[] obaArrivalInfoArray = new ObaArrivalInfo[1];
-        obaArrivalInfoArray[0] = obaArrivalInfo;
-        new Expectations() {{
-            obaArrivalInfo.getShortName(); result = "8";
-            obaArrivalInfo.getHeadsign(); result = "Mlk Way Jr";
-            obaArrivalInfoResponse.getArrivalInfo(); result = obaArrivalInfoArray;
-            obaUserClient.getArrivalsAndDeparturesForStop(anyString, anyInt); result = obaArrivalInfoResponse;
-        }};
-
-        SpeechletResponse sr = authedSpeechlet.onLaunch(
-                launchRequest,
-                session);
-
-        String spoken = ((PlainTextOutputSpeech)sr.getOutputSpeech()).getText();
-        assertThat(spoken, equalTo("Route 8 Mlk Way Jr is departing now based on the schedule -- "));
```

Comment:
Could you point me to where the new unit tests are for testing the get arrivals intent? Unless I'm missing something it seems like there is a significant decrease in test coverage in this PR compared to the master branch.

Reply:
I only migrated part of the test cases because of the timeline, will add more UTs in the following pull request, Thanks

Reply:
Sounds good. I'm ok with merging this PR without full unit test coverage as long as you commit to having unit test parity with the current master branch in another pull request.